### ISSUE TITLE
not an issue but suggestion about ckpt naming

### ISSUE THREAD

**Issue by 1blackbar** [TheLastBen/fast-stable-diffusion#230]
I do a lot of trainings and it would be really handy to save models by - instancename-class-stepsnumber-imagesnumber-learningrate.ckpt

So its like "sks-man-1600-25-1e-6.ckpt"

I test various settings and its just easier this way to compare them and pick the best settings.

**Comment by TheLastBen** [comment]
I'll add it

**Comment by TheLastBen** [comment]
avoid using sks as an instance name, SD knows it's a gun

**Comment by giteeeeee** [comment]
> avoid using sks as an instance name, SD knows it's a gun

I had an instance named "a100". Then it was trained on a person...

Glad I didn't see any output resembling [person|graphics card]

**Comment by TheLastBen** [comment]
but it could mess the coherence

**Comment by 1blackbar** [comment]
yeah it was just example , before i use instance name i type it into SD to see if it exists
